api/src/main/resources/liquibase.xml

```diff
@@ -1020,6 +1020,32 @@ VALUES
             WHERE pp.program_id IN (5,6,7,8,9);
         </sql>
     </changeSet>
+
+	<changeSet id="ugandaemr-20240610-1655" author="slubwama">
+		<preConditions onFail="CONTINUE">
+			<sqlCheck expectedResult="1">
+				select count(*) from concept where uuid=="7cac8397-53cd-4f00-a6fe-028e8d743f8e";
```

Comment:
Wrong MySQL syntax `select count(*) from concept where uuid=="7cac8397-53cd-4f00-a6fe-028e8d743f8e";`

Should be `select count(*) from concept where uuid="7cac8397-53cd-4f00-a6fe-028e8d743f8e";`

Single equal...